docs/install/ubuntu_setup.md

```diff
@@ -92,7 +92,7 @@ The following table presents the pip packages that are recommended for each vers
 | mxnet | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 -->
 
-![pip package table](https://user-images.githubusercontent.com/5974205/42119928-362ad5ba-7bc7-11e8-97de-dba8fd099c90.png)
+![pip package table](https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages.png)
```

Comment:
we don't intend to provide 1.2.1 pip package because of its breakage. 1.2.1 should be changed to 1.2.0.

Reply:
Ok I updated the image. It should appear if you hard-refresh.
https://github.com/aaronmarkham/web-data/blob/ae8a7beff6dcf5594a74791242a3ebde796aab8e/mxnet/install/pip-packages.png